Test/Unit/Model/Import/RuleModelTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Taxjar\SalesTax\Test\Unit\Model\Import;
+
+use Magento\Framework\Api\AttributeValueFactory;
+use Magento\Framework\Api\ExtensionAttributesFactory;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Framework\Event\ManagerInterface;
+use Magento\Framework\Model\Context;
+use Magento\Framework\Model\ResourceModel\AbstractResource;
+use Magento\Framework\Registry;
+use Magento\Tax\Model\Calculation;
+use Magento\Tax\Model\Calculation\Rule\Validator;
+use Magento\Tax\Model\ClassModel;
+use Taxjar\SalesTax\Model\Import\RuleModel;
+use Taxjar\SalesTax\Test\Unit\UnitTestCase;
+
+class RuleModelTest extends UnitTestCase
+{
+    public function testAfterSave()
```

Comment:
Could a more descriptive name better indicated what result is being test. Maybe something like testAfterSaveDispatchesTaxSettingChangeAfter. That might be a little to verbose though?